examples/express-ex/index.js

```diff
@@ -1,8 +1,8 @@
-/* eslint-disable no-console */
+/* eslint-disable */
 
 const express = require("express");
-const {Receiver} = require("cloudevents");
```

Comment:
Should we wait until the 3.2.0 release comes out before updating this example?

Reply:
Hmm - I suppose. I actually didn't mean to commit that. I was just using it to test out the new interfaces. I'm good either way. I can revert this file.